src/JointAxis.cc

```diff
@@ -374,37 +375,54 @@ sdf::ElementPtr JointAxis::Element() const
   return this->dataPtr->sdf;
 }
 
-/////////////////////////////////////////////////
 sdf::ElementPtr JointAxis::ToElement(unsigned int _index) const
+{
+  sdf::Errors errors;
+  auto result = this->ToElement(errors, _index);
+  sdf::throwOrPrintErrors(errors);
+  return result;
+}
+
+/////////////////////////////////////////////////
+sdf::ElementPtr JointAxis::ToElement(sdf::Errors &_errors,
+                                     unsigned int _index) const
 {
   sdf::ElementPtr elem(new sdf::Element);
   sdf::initFile("joint.sdf", elem);
 
   std::string axisElemName = "axis";
   if (_index > 0u)
     axisElemName += std::to_string(_index + 1);
-  sdf::ElementPtr axisElem = elem->GetElement(axisElemName);
-  sdf::ElementPtr xyzElem = axisElem->GetElement("xyz");
-  xyzElem->Set<gz::math::Vector3d>(this->Xyz());
+  sdf::ElementPtr axisElem = elem->GetElement(axisElemName, _errors);
+  sdf::ElementPtr xyzElem = axisElem->GetElement("xyz", _errors);
+  xyzElem->Set<gz::math::Vector3d>(_errors, this->Xyz());
   if (!this->XyzExpressedIn().empty())
   {
     xyzElem->GetAttribute("expressed_in")->Set<std::string>(
-        this->XyzExpressedIn());
+        this->XyzExpressedIn(), _errors);
```

Comment:
Ah coverage is complaining about this line. Perhaps testing with `JointAxis::SetXyzExpressedIn` would make it go away